### PR TITLE
added pageWaitFor and cookies to options. Changed security for webfonts

### PR DESCRIPTION
**Pull request by fduarte42**
added page WaitFor timeout (in miliseconds)
added --disable-web-security to launch so web-fonts can be loaded

### CODE REVIEW

**Comment by fduarte42** [comment]
Hi, I wanted to ask, if there is something missing for a merge. Or if I should change something.

Thanks in advance.

File: lib/Converter.js [hunk 1]
```diff
@@ -74,7 +74,16 @@ class Converter
         }
 
         return this._setHtml(page, this.html)
-            .then(() => this._getPdf(page, options));
+            .then(() => {
+                if (options.hasOwnProperty('pageWaitFor')) {
+                    let timeout = parseInt(options.pageWaitFor);
```
Comment (spiritix) [inline lib/Converter.js]: this should be const not let

File: lib/Converter.js [hunk 2]
```diff
@@ -74,7 +74,16 @@ class Converter
...
+                    delete options.pageWaitFor;
+                    return page.waitFor(timeout).then(() => this._getPdf(page, options));
+                } else {
+                    return this._getPdf(page, options)
+                }
+
```
Comment (spiritix) [inline lib/Converter.js]: remove this NL

**Comment by spiritix** [comment]
Sorry, I'm a bit short in time currently. Just two small comments, after that I will merge it.

**Pull request merged**